lib/main.js

```diff
@@ -44,8 +44,13 @@ function accumulator({
   return calculateAccumulator(selections, stake, ew, idx, place);
 }
 
-function calculateMultiple(selections, stake, ew = false, fullCover = false) {
-  const count = selections.length;
+function calculateMultiple(
+  selections,
+  stake,
+  ew = false,
+  fullCover = false,
+  count = selections.length
```

Comment:
How come this has moved from inside the function to outside of it?

Reply:
there was a suggestion from Ollie.
so we are passing a new variable size

quoting from Ollie:

```function calculateMultiple(selections, stake, ew = false, fullCover = false, size = null) {
      const count = size || selections.length;
```
to:
```
function calculateMultiple(selections, stake, ew = false, fullCover = false, count = selections.length) {
```

I thought it's a good idea. What do you think?

Reply:
Ah okay, I get that.

My concern isn't so much with the implementation, it's more the arity creep. So we went from `calculateMultiple/4` to `calculateMultiple/5` and admittedly some are default variables.

Would we know the values when they appear in the `multiple` function? 

If so we could use this to work out whether it needs default values or not then we can pass in one argument as an object to this function.